### PR TITLE
Add multiple improvements

### PR DESCRIPTION
**Pull request by ThishaniLucas**
* Make package name mandatory for package gen.
* Make dependent package mandatory for template gen.
* Rename --ig-package to --dependent-package
* Remove --dependency option and its initialization
* Update help text

### CODE REVIEW

File: native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/config/BallerinaProjectToolConfig.java [hunk 1]
```diff
@@ -207,6 +194,6 @@ public String getServicePackage() {
         return servicePackage;
     }
     public String getResourcePackage() {
```
Comment (sameeragunarathne) [inline native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/config/BallerinaProjectToolConfig.java:196]:
```suggestion

    public String getDependentPackage() {
```

Change the usages too

Comment (ThishaniLucas) [reply native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/config/BallerinaProjectToolConfig.java]: done

File: native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java [hunk 2]
```diff
@@ -86,10 +87,8 @@ public class FhirSubCmd implements BLauncherCmd {
     @CommandLine.Option(names = "--excluded-profile", description = "Profiles to be excluded in the template")
     private String[] excludedProfiles;
 
-    @CommandLine.Option(names = "--ig-package", description = "Resource package name for the templates to be generated")
-    private String resourcePackage;
-    @CommandLine.Option(names = "--dependency", converter = JsonTypeConverter.class, description = "custom dependency to be added")
-    private JsonElement dependency;
+    @CommandLine.Option(names = "--dependent-package", description = "Resource package name for the templates to be generated")
```
Comment (sameeragunarathne) [inline native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java:90]: description needs to be changed?

Comment (ThishaniLucas) [reply native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java]: updated

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**